www/shia02/du3_styles.css

```diff
@@ -54,18 +59,19 @@
 }
 
 .box-03 {
-    margin-bottom: 10px;
+    width: 100%;
```

Comment:
chybí box-sizing: border-box, a proto se stalo něco nečekaného u box-03 - jeho šířka je větší než šířka stránky, přestože ta má být 100%, je to protože ještě tam je šířka paddingu a borderu, použití box-sizing: border-box započte padding a border do width. Tahle vlastnost je dobré dávat do všech elementů pomocí wildcard selektoru (hvězdička)